src/Util/LocaleCodeProvider.php

```diff
@@ -67,16 +65,31 @@ public function getFormattedLocaleCode(string $localeCode): string
     {
         $canonicalizedCode = (string) \Locale::canonicalize($localeCode);
 
-        if (\mb_strlen($canonicalizedCode) !== self::SUPPORTED_LOCALE_CODE_LENGTH) {
+        $locales = (new \ReflectionClass(PaypalLocales::class))->getConstants();
```

Comment:
I'm not a fan of using Reflection here. Could we maybe add a constant to that class, that holds all countries? Or a two-dimensional array?

Reply:
Why are you not a fan? Having the the locales as an array is for me worse than this.